packages/elastic-apm-synthtrace/src/lib/utils/clean_write_targets.ts

```diff
@@ -27,6 +27,7 @@ export async function cleanWriteTargets({
     index: targets,
     allow_no_indices: true,
     conflicts: 'proceed',
+    refresh: true,
```

Comment:
@dgieselaar Merging this with https://github.com/elastic/kibana/pull/116770 was a little tricky, does this look right?

Reply:
Yeah, looks fine!